server/routes/bookedtoday/summaryController.ts

```diff
@@ -0,0 +1,45 @@
+import type { RequestHandler } from 'express'
+import { ExpectedArrivalsService } from '../../services'
+import { State } from './arrivals/state'
+import { MatchType } from '../../services/matchTypeDecorator'
+
+export default class SummaryController {
+  public constructor(private readonly expectedArrivalsService: ExpectedArrivalsService) {}
+
+  public view(): RequestHandler {
+    return async (req, res) => {
+      const { id } = req.params
+      const arrival = await this.expectedArrivalsService.getArrival(id)
+      return res.render('pages/bookedtoday/summary.njk', { arrival })
+    }
+  }
+
+  public redirectToConfirm(): RequestHandler {
+    return async (req, res) => {
```

Comment:
Talked to @louisenorris about this we've agreed to temporarily duplicate the `choosePrisonerController` logic here whilst this work is still in progress, and the Summary Page is not integrated into the UX.

Not 100% clear whether this logic will exist only here once the Summary Page is integrated or if there are scenarios where the user will go straight to confirming an arrival without a summary.

Either way, we can either refactor or remove the logic in `choosePrisonerController` when we're crossing that bridge 😄 

Happy to hear third opinions on this, though! 🧠